server.go

```diff
@@ -377,6 +380,49 @@ func main() {
 	}
 	fmt.Println("WorkspaceGateway provided")
 
+	// Provide OpenSearch Client
+	err = c.Provide(func() (*opensearch.Client, error) {
```

Comment:
We don't keep the creation of the client here in server.go, move to config module.

Reply:
Yes moving it there